public/src/managers/CourseManager.ts

```diff
@@ -224,38 +224,19 @@ export class CourseManager {
         return null;
     }
 
-    public async getGroupCourseForTeacher(group: Group, course: Course, assignments: Assignment[]):
```

Comment:
Is this obsoleted?

Reply:
It was used to retrieve all group lab submissions before, now we have another method that fetches individual or group labs depending on the context.